test/offline/test_cql.py

```diff
@@ -185,14 +185,6 @@ def save_best_fn(policy):
     def stop_fn(mean_rewards):
         return mean_rewards >= args.reward_threshold
 
-    def watch():
```

Comment:
Why this change? Was the function unused?

Reply:
Yes it was unused